modules/serverless/variables.tf

```
@@ -32,6 +32,12 @@ variable "auto_stop_configuration" {
   default     = {}
 }
 
+variable "image_configuration" {
+  description = "The custom image configuration applied to all worker types"
```

Copied from the provider
```suggestion
  description = "The image configuration applied to all worker types"
```

---

done